packages/relay-signer/src/relayer.ts

```diff
@@ -18,6 +23,23 @@ export class Relayer implements IRelayer {
   private relayer: IRelayer;
   private credentials: RelayerParams;
 
+  private isEthersV5Provider(
+    _provider: JsonRpcProvider | Provider,
+    ethersVersion?: EthersVersion,
+  ): _provider is Provider {
+    // default to ethers v5
+    if (!ethersVersion || ethersVersion === 'v5') return true;
+    return false;
+  }
+
+  private isEthersV5ProviderOptions(
+    options?: DefenderRelaySignerOptionsV5 | DefenderRelaySignerOptions,
+  ): options is DefenderRelaySignerOptionsV5 {
+    // default to ethers v5
```

Comment:
Whenever we switch our ethers dependency in action layers, we should also release a SDK version that defaults to ethers v6